### ISSUE TITLE
EventSource with EventCounter

### ISSUE THREAD

**Issue by shacal** [Azure/diagnostics-eventflow#323]
What output is this?
`{metricNameProperty: "Mean", metricValueProperty: "Mean"}`

Kinda sucks when internal libraries does things badly.

And for testing
```
var eventCounterMetric = new EventMetadata(MetricData.MetricMetadataKind);
eventCounterMetric.Properties.Add("metricNameProperty", "EventCounterValue");
eventCounterMetric.Properties.Add("metricValueProperty", "EventCounterValue");

eventData.SetMetadata(eventCounterMetric);
eventData.Payload.Add("EventCounterValue", 1d);

eventData.TryGetMetadata(MetricData.MetricMetadataKind, out var metadataCollection);
foreach (var metricData in metadataCollection)
{
    MetricData.TryGetData(eventData, metricData, out var metric);
    Assert.Equal(metric.MetricName, "EventCounterValue");
    Assert.Equal(metric.Value, 1d);
}
```

Did someone forget these?

```
public class MetricData
public static readonly string MetricMetadataKind = "metric";
public static readonly string MetricNameMoniker = "metricName";
public static readonly string MetricNamePropertyMoniker = "metricNameProperty";
public static readonly string MetricValueMoniker = "metricValue";
public static readonly string MetricValuePropertyMoniker = "metricValueProperty";
```

**Comment by shacal** [comment]
https://github.com/Azure/diagnostics-eventflow/blob/6d8bfbfed4516ec584a5c75f97975b9d23d1e9bc/src/Microsoft.Diagnostics.EventFlow.Inputs.EventSource/EventDataExtensions.cs#L104

I suggest that this will be `MetricData.MetricNamePropertyMoniker` changed to `MetricData.MetricNameMoniker`

**Comment by karolz-ms** [comment]
@shacal thank you for reporting this issue!

I have just published EventSource input version 1.4.6 on Nuget; that version should fix the problem.
